src/md/community.md

```diff
@@ -1,6 +1,6 @@
-# Community
+# <a name="community"></a>Community
```

Comment:
Thanks. I removed these anchors in md files and used the implicit id generated by Markdown.